### DISCUSSION TITLE
Mesh adaptation in V7.1.1 to resolve shocks

### DISCUSSION THREAD

**Discussion by ashvinmahajan** [su2code/SU2#1246] [Q&A]
Hello,
Has anyone been able to use the mesh adaptation options in V7.1.1? I would like to resolve the mesh in the vicinity of a shock without having to manually change the mesh resolution for each angle of attack.  Are these options still valid in V7.1.1? 
In the config template file there are options:
% ------------------------- GRID ADAPTATION STRATEGY --------------------------%
%
% Kind of grid adaptation (NONE, PERIODIC, FULL, FULL_FLOW, GRAD_FLOW,
% FULL_ADJOINT, GRAD_ADJOINT, GRAD_FLOW_ADJ, ROBUST,
% FULL_LINEAR, COMPUTABLE, COMPUTABLE_ROBUST,
% REMAINING, WAKE, SMOOTHING, SUPERSONIC_SHOCK)
KIND_ADAPT= FULL_FLOW
%
% Percentage of new elements (% of the original number of elements)
NEW_ELEMS= 5
%
% Scale factor for the dual volume
DUALVOL_POWER= 0.5
%
% Adapt the boundary elements (NO, YES)
ADAPT_BOUNDARY= YES

Thanks
Ashvin

**Comment by pcarruscag** [comment]
Hi Ashvin,
There is no native mesh adaptation in SU2 v7 anymore.
Search the issues for "mesh adaptation" and you will find information about the "feature_adap_????" branches, which I think some folks other than the developer have used successfully.
Cheers,
Pedro
